slides/12_Styling.html

```diff
@@ -1,178 +1,43 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
-    />
 
-    <title>Styling | Essential React</title>
+<head>
```

Comment:
This indenting seems off. Is Prettier not running on this file when saving? It fails to run in case the nesting is broken in the HTML file. The error message is then shown in the Output of Prettier in VS Code